src/main/java/io/reactivex/disposables/SubscriptionDisposable.java

```diff
@@ -15,6 +15,9 @@
 import org.reactivestreams.Subscription;
 
 final class SubscriptionDisposable extends ReferenceDisposable<Subscription> {
+    /** */
+    private static final long serialVersionUID = -707001650852963139L;
```

Comment:
What are these for? We shouldn't be encouraging people to serialize these instances.